assets/js/components/HostDetails/HostSettingsPage.jsx

```diff
@@ -29,6 +29,8 @@ function HostSettingsPage() {
       setSelection(hostSelectedChecks);
     }
   }, [hostSelectedChecks]);
+  const [checksExecutionTooltipVisible, setChecksExecutionTooltipVisible] =
```

Comment:
`hostSelectedChecks` should update itself reactively if I remember correctly, did you check if you actually need a state here?

Reply:
Good point :+1: removed this